referee.c

```diff
@@ -0,0 +1,180 @@
+#include <stdio.h>
+#include <time.h>
+#include <Windows.h>
+#include <stdlib.h>
+#include "referee.h"
+
+//#define N 3
+const char markerX = 'X';

```

Comment:
I think these constants must be defined in a common header file that must be provided not only to the referee, but also to each algorithm.